### ISSUE TITLE
Be careful this version (WART-1.8.1.0.exe) from wart is virus !!!

### ISSUE THREAD

**Issue by AhmedRaffat** [venomous0x/WhatsAPI#1137]
Be careful this version (WART-1.8.1.0.exe) from wart is virus !!!
https://github.com/mgp25/WART

**Comment by matteocontrini** [comment]
LOL

**Comment by mgp25** [comment]
How bad is the envy, the source code its public, you can check it and if you know a little of programming, you will see everything its clearly done.

You need to learn more

Best regards

**Comment by AhmedRaffat** [comment]
no no no source code is not public and you know that you only add .exe file without source code (WhatsAppApi.dll) is missing and old version you only add .exe file without update source code

You need to learn more

Best regards

**Comment by thiagosantos** [comment]
This is true, the WhatsAppApi.dll is already compiled and it source's code it's not available in this repo.

@AhmedRaffat What kind of virus warning did you receive?

**Comment by AhmedRaffat** [comment]
@thiagosantos Trojan.Win32.Generic! - Win32: Malware-gen 
please see this link : https://www.virustotal.com/ru/file/78378d686853cf1a089bada07966483ab470c3fb840b6e53035fca0fd3c1882c/analysis/

@mgp25 play with us and block me

**Comment by AhmedRaffat** [comment]
i need to ask all developer use @mgp25 API have you ever thought what @mgp25 winning from this business
LOL Very soon all we will be known

Best regards

**Comment by mgp25** [comment]
All my stuff its public, not like the business you own.

The whatsapi.dll is just the fucking whatsapinet compiled. And if you are curious enough use a fucking decompiler and check the fucking source code

**Comment by AhmedRaffat** [comment]
@mgp25 please be cool

**Comment by thiagosantos** [comment]
@mgp25 This is sad but, apparently true, I've checked all WART-1.*.*.*.exe. The lasts two versions, on Avast, (WART-1.8.*.0.exe) throws me a warning. This can also be a false-positive as this one https://forum.avast.com/index.php?topic=147249.0 .

Details:
----------------------------------------------
URL: https://raw.githubusercontent.com/mgp25/WART/master/Old/WART-1.8.0.0.exe
Infecção: FileRepMetagen [Malware]
----------------------------------------------
URL: https://raw.githubusercontent.com/mgp25/WART/master/WART-1.8.1.0.exe
Infecção: Win32:Malware-gen

Hope be useful.

@AhmedRaffat anyway you can try compile the DLL  and WART by yourself. Good look.

**Comment by mgp25** [comment]
@Thiagosantos why dont you check the source code and proof yourselve that its a false positive. 

Check it

**Comment by khaledway** [comment]
@mgp25  I try to download the source code and i cant , can you help me and provide direct link to the source code ?
